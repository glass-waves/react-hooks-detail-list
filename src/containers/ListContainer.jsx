import List from '../components/list/List'
import { useFetch } from '../hooks/useFetch'
import React from 'react';

const ListContainer = () => {

    const { cardList, loading} = useFetch(); 

    if (loading) return <h1>Loading</h1>;

    return (
        <div>
            <List cards={cardList}/> 
        </div>
    )
}

export default ListContainer
