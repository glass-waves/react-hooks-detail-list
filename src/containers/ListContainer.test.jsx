import React from 'react';
import { render, screen, waitFor } from '@testing-library/react';
import ListContainer from './ListContainer';
import { rest } from 'msw';
import { setupServer } from 'msw/node';
import data from '../services/data.json';

const server = setupServer(
    rest.get('https://api.magicthegathering.io/v1/cards', (req, res, ctx) => {
        return res(ctx.json(data))
    })
);


describe('ListContainer', () => {
    beforeAll(() => server.listen());
    afterAll(() => server.close());

    it('renders a list of magic cards to the page', async () => {
        render(<ListContainer />);
        screen.getByText('Loading');
        const ul = await screen.findByRole('list', { name: 'cards' });
        expect(ul).toMatchSnapshot();
    });
});
