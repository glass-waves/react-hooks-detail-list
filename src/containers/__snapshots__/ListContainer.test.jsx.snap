// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ListContainer renders a list of magic cards to the page 1`] = `
<ul
  aria-label="cards"
>
  <a
    href="/5f8287b1-5bb6-5f4c-ad17-316a40d5bb0c"
  >
    <li>
      <h1>
        Ancestor's Chosen
      </h1>
      <img
        alt="Ancestor's Chosen"
        src="http://gatherer.wizards.com/Handlers/Image.ashx?multiverseid=130550&type=card"
      />
    </li>
  </a>
  <a
    href="/57aaebc1-850c-503d-9f6e-bb8d00d8bf7c"
  >
    <li>
      <h1>
        Angel of Mercy
      </h1>
      <img
        alt="Angel of Mercy"
        src="http://gatherer.wizards.com/Handlers/Image.ashx?multiverseid=129465&type=card"
      />
    </li>
  </a>
  <a
    href="/8ac972b5-9f6e-5cc8-91c3-b9a40a98232e"
  >
    <li>
      <h1>
        Aven Cloudchaser
      </h1>
      <img
        alt="Aven Cloudchaser"
        src="http://gatherer.wizards.com/Handlers/Image.ashx?multiverseid=129470&type=card"
      />
    </li>
  </a>
  <a
    href="/a69b404f-144a-5317-b10e-7d9dce135b24"
  >
    <li>
      <h1>
        Ballista Squad
      </h1>
      <img
        alt="Ballista Squad"
        src="http://gatherer.wizards.com/Handlers/Image.ashx?multiverseid=129477&type=card"
      />
    </li>
  </a>
  <a
    href="/6d268c95-c176-5766-9a46-c14f739aba1c"
  >
    <li>
      <h1>
        Bandage
      </h1>
      <img
        alt="Bandage"
        src="http://gatherer.wizards.com/Handlers/Image.ashx?multiverseid=132106&type=card"
      />
    </li>
  </a>
  <a
    href="/56f4935b-f6c5-59b9-88bf-9bcce20247ce"
  >
    <li>
      <h1>
        Beacon of Immortality
      </h1>
      <img
        alt="Beacon of Immortality"
        src="http://gatherer.wizards.com/Handlers/Image.ashx?multiverseid=130553&type=card"
      />
    </li>
  </a>
  <a
    href="/62365fbb-af07-5c2a-976d-e3092bdfc317"
  >
    <li>
      <h1>
        Benalish Knight
      </h1>
      <img
        alt="Benalish Knight"
        src="http://gatherer.wizards.com/Handlers/Image.ashx?multiverseid=136279&type=card"
      />
    </li>
  </a>
  <a
    href="/0d2a4e1d-cc12-5675-8044-9a7bc7d60050"
  >
    <li>
      <h1>
        Benalish Knight
      </h1>
      <img
        alt="Benalish Knight"
      />
    </li>
  </a>
  <a
    href="/81daea6a-2735-5a46-a2da-b65a2ad5738f"
  >
    <li>
      <h1>
        Cho-Manno, Revolutionary
      </h1>
      <img
        alt="Cho-Manno, Revolutionary"
        src="http://gatherer.wizards.com/Handlers/Image.ashx?multiverseid=130554&type=card"
      />
    </li>
  </a>
  <a
    href="/7fef665c-36a1-5f7a-9299-cf8938708710"
  >
    <li>
      <h1>
        Condemn
      </h1>
      <img
        alt="Condemn"
        src="http://gatherer.wizards.com/Handlers/Image.ashx?multiverseid=130528&type=card"
      />
    </li>
  </a>
  <a
    href="/9c0311e8-03c1-527d-9d90-6fa2dba2aae9"
  >
    <li>
      <h1>
        Contaminated Bond
      </h1>
      <img
        alt="Contaminated Bond"
      />
    </li>
  </a>
  <a
    href="/d9902515-6292-5a4a-9bde-a7616ad5f0bc"
  >
    <li>
      <h1>
        Drudge Skeletons
      </h1>
      <img
        alt="Drudge Skeletons"
      />
    </li>
  </a>
  <a
    href="/2f9c211e-1869-5b3f-94ea-f73b7910a5af"
  >
    <li>
      <h1>
        Demystify
      </h1>
      <img
        alt="Demystify"
        src="http://gatherer.wizards.com/Handlers/Image.ashx?multiverseid=129524&type=card"
      />
    </li>
  </a>
  <a
    href="/f49300ef-efa0-5b82-ace3-f2f06795fbd5"
  >
    <li>
      <h1>
        Fear
      </h1>
      <img
        alt="Fear"
      />
    </li>
  </a>
  <a
    href="/668c64a2-cecb-5f48-af16-d7a64814d3e7"
  >
    <li>
      <h1>
        Field Marshal
      </h1>
      <img
        alt="Field Marshal"
        src="http://gatherer.wizards.com/Handlers/Image.ashx?multiverseid=135258&type=card"
      />
    </li>
  </a>
  <a
    href="/f39b4446-0ca6-5446-8507-4b1ba37bb3e7"
  >
    <li>
      <h1>
        Hypnotic Specter
      </h1>
      <img
        alt="Hypnotic Specter"
      />
    </li>
  </a>
  <a
    href="/d68306e2-9877-5987-84b3-12b8234c8eec"
  >
    <li>
      <h1>
        Ghost Warden
      </h1>
      <img
        alt="Ghost Warden"
        src="http://gatherer.wizards.com/Handlers/Image.ashx?multiverseid=132105&type=card"
      />
    </li>
  </a>
  <a
    href="/546eac7c-1424-597d-ac13-bf8558e88fe3"
  >
    <li>
      <h1>
        Glorious Anthem
      </h1>
      <img
        alt="Glorious Anthem"
        src="http://gatherer.wizards.com/Handlers/Image.ashx?multiverseid=129572&type=card"
      />
    </li>
  </a>
  <a
    href="/c19bcfc2-fc10-5040-8239-1c193098df47"
  >
    <li>
      <h1>
        Hail of Arrows
      </h1>
      <img
        alt="Hail of Arrows"
        src="http://gatherer.wizards.com/Handlers/Image.ashx?multiverseid=132107&type=card"
      />
    </li>
  </a>
  <a
    href="/36fcaa1e-64e3-56e3-950c-907db167e41f"
  >
    <li>
      <h1>
        Heart of Light
      </h1>
      <img
        alt="Heart of Light"
        src="http://gatherer.wizards.com/Handlers/Image.ashx?multiverseid=132090&type=card"
      />
    </li>
  </a>
  <a
    href="/60f49caf-3583-5f85-b4b3-08dca73a8628"
  >
    <li>
      <h1>
        High Ground
      </h1>
      <img
        alt="High Ground"
        src="http://gatherer.wizards.com/Handlers/Image.ashx?multiverseid=132145&type=card"
      />
    </li>
  </a>
  <a
    href="/9bee1901-1125-5635-9e22-3b4f37989c37"
  >
    <li>
      <h1>
        Honor Guard
      </h1>
      <img
        alt="Honor Guard"
        src="http://gatherer.wizards.com/Handlers/Image.ashx?multiverseid=129595&type=card"
      />
    </li>
  </a>
  <a
    href="/cb865391-5b14-58ce-aa99-b36b83e6377f"
  >
    <li>
      <h1>
        Icatian Priest
      </h1>
      <img
        alt="Icatian Priest"
        src="http://gatherer.wizards.com/Handlers/Image.ashx?multiverseid=132123&type=card"
      />
    </li>
  </a>
  <a
    href="/39b50224-8305-56dd-9849-09deecb07d20"
  >
    <li>
      <h1>
        Reviving Dose
      </h1>
      <img
        alt="Reviving Dose"
        src="http://gatherer.wizards.com/Handlers/Image.ashx?multiverseid=132089&type=card"
      />
    </li>
  </a>
  <a
    href="/6028cab2-0478-584b-a880-8174a88eb188"
  >
    <li>
      <h1>
        Reya Dawnbringer
      </h1>
      <img
        alt="Reya Dawnbringer"
        src="http://gatherer.wizards.com/Handlers/Image.ashx?multiverseid=106384&type=card"
      />
    </li>
  </a>
  <a
    href="/9a14261a-b567-5429-a5ca-a913e15d8bf7"
  >
    <li>
      <h1>
        Righteousness
      </h1>
      <img
        alt="Righteousness"
        src="http://gatherer.wizards.com/Handlers/Image.ashx?multiverseid=130552&type=card"
      />
    </li>
  </a>
  <a
    href="/a2be2e9b-5211-53e6-ac6a-abd65dbcac1d"
  >
    <li>
      <h1>
        Samite Healer
      </h1>
      <img
        alt="Samite Healer"
        src="http://gatherer.wizards.com/Handlers/Image.ashx?multiverseid=132101&type=card"
      />
    </li>
  </a>
  <a
    href="/fe9187fa-cd14-5618-9a62-bcf18b523d84"
  >
    <li>
      <h1>
        Serra's Embrace
      </h1>
      <img
        alt="Serra's Embrace"
        src="http://gatherer.wizards.com/Handlers/Image.ashx?multiverseid=135214&type=card"
      />
    </li>
  </a>
  <a
    href="/b8a68840-4044-52c0-a14e-0a1c630ba42c"
  >
    <li>
      <h1>
        Academy Researchers
      </h1>
      <img
        alt="Academy Researchers"
        src="http://gatherer.wizards.com/Handlers/Image.ashx?multiverseid=132072&type=card"
      />
    </li>
  </a>
  <a
    href="/76ddd7f5-1a84-55d5-98f5-4883c217e0d8"
  >
    <li>
      <h1>
        Air Elemental
      </h1>
      <img
        alt="Air Elemental"
        src="http://gatherer.wizards.com/Handlers/Image.ashx?multiverseid=129459&type=card"
      />
    </li>
  </a>
  <a
    href="/29723d8d-34c9-5009-8b79-2c679e35f674"
  >
    <li>
      <h1>
        Ambassador Laquatus
      </h1>
      <img
        alt="Ambassador Laquatus"
        src="http://gatherer.wizards.com/Handlers/Image.ashx?multiverseid=129913&type=card"
      />
    </li>
  </a>
  <a
    href="/758df5d5-992f-5d30-b6bc-87cb1a0788ab"
  >
    <li>
      <h1>
        Aven Fisher
      </h1>
      <img
        alt="Aven Fisher"
        src="http://gatherer.wizards.com/Handlers/Image.ashx?multiverseid=130985&type=card"
      />
    </li>
  </a>
  <a
    href="/d833c453-58c5-5ff6-91b2-f526ca5f808a"
  >
    <li>
      <h1>
        Aven Windreader
      </h1>
      <img
        alt="Aven Windreader"
        src="http://gatherer.wizards.com/Handlers/Image.ashx?multiverseid=129473&type=card"
      />
    </li>
  </a>
  <a
    href="/1aac00c8-3b6e-5bdd-9fde-6cfcbc7837c0"
  >
    <li>
      <h1>
        Boomerang
      </h1>
      <img
        alt="Boomerang"
        src="http://gatherer.wizards.com/Handlers/Image.ashx?multiverseid=129494&type=card"
      />
    </li>
  </a>
  <a
    href="/cdc96814-2f52-5314-80ab-0cff2dcc5054"
  >
    <li>
      <h1>
        Cancel
      </h1>
      <img
        alt="Cancel"
        src="http://gatherer.wizards.com/Handlers/Image.ashx?multiverseid=129882&type=card"
      />
    </li>
  </a>
  <a
    href="/9af574e4-dc6e-5246-8fe7-fee6c4c77a76"
  >
    <li>
      <h1>
        Cephalid Constable
      </h1>
      <img
        alt="Cephalid Constable"
        src="http://gatherer.wizards.com/Handlers/Image.ashx?multiverseid=135261&type=card"
      />
    </li>
  </a>
  <a
    href="/8dfc67e9-8323-5d1f-9e25-9f9394abd5a0"
  >
    <li>
      <h1>
        Clone
      </h1>
      <img
        alt="Clone"
        src="http://gatherer.wizards.com/Handlers/Image.ashx?multiverseid=129501&type=card"
      />
    </li>
  </a>
  <a
    href="/596a7bd4-895b-5e7c-8124-373fd709444d"
  >
    <li>
      <h1>
        Cloud Elemental
      </h1>
      <img
        alt="Cloud Elemental"
        src="http://gatherer.wizards.com/Handlers/Image.ashx?multiverseid=129804&type=card"
      />
    </li>
  </a>
  <a
    href="/fdbec3b3-f6f2-5149-a70a-4f9d2622a87b"
  >
    <li>
      <h1>
        Cloud Sprite
      </h1>
      <img
        alt="Cloud Sprite"
        src="http://gatherer.wizards.com/Handlers/Image.ashx?multiverseid=132069&type=card"
      />
    </li>
  </a>
  <a
    href="/eb5c94c7-5494-5669-9c54-6f3672d65683"
  >
    <li>
      <h1>
        Counsel of the Soratami
      </h1>
      <img
        alt="Counsel of the Soratami"
        src="http://gatherer.wizards.com/Handlers/Image.ashx?multiverseid=134757&type=card"
      />
    </li>
  </a>
  <a
    href="/8d5f87fd-7a87-572b-bdce-303effec6b92"
  >
    <li>
      <h1>
        Crafty Pathmage
      </h1>
      <img
        alt="Crafty Pathmage"
        src="http://gatherer.wizards.com/Handlers/Image.ashx?multiverseid=134758&type=card"
      />
    </li>
  </a>
  <a
    href="/5a50a7e8-2ed1-5e8a-8867-c96f1186bd47"
  >
    <li>
      <h1>
        Dehydration
      </h1>
      <img
        alt="Dehydration"
        src="http://gatherer.wizards.com/Handlers/Image.ashx?multiverseid=129521&type=card"
      />
    </li>
  </a>
  <a
    href="/a9a74d88-7663-5419-a74c-5a28e8642b4f"
  >
    <li>
      <h1>
        Deluge
      </h1>
      <img
        alt="Deluge"
        src="http://gatherer.wizards.com/Handlers/Image.ashx?multiverseid=135223&type=card"
      />
    </li>
  </a>
  <a
    href="/f4ee3e9e-21da-5a63-8278-3efa59da7df3"
  >
    <li>
      <h1>
        Denizen of the Deep
      </h1>
      <img
        alt="Denizen of the Deep"
        src="http://gatherer.wizards.com/Handlers/Image.ashx?multiverseid=135250&type=card"
      />
    </li>
  </a>
  <a
    href="/e2066f75-65de-5254-82f2-8e1f969bf31b"
  >
    <li>
      <h1>
        Discombobulate
      </h1>
      <img
        alt="Discombobulate"
        src="http://gatherer.wizards.com/Handlers/Image.ashx?multiverseid=134749&type=card"
      />
    </li>
  </a>
  <a
    href="/6f9ae90a-5638-5b88-a3d7-ccea4372eb79"
  >
    <li>
      <h1>
        Dreamborn Muse
      </h1>
      <img
        alt="Dreamborn Muse"
        src="http://gatherer.wizards.com/Handlers/Image.ashx?multiverseid=135246&type=card"
      />
    </li>
  </a>
  <a
    href="/100f46ae-e457-5491-b5d1-66cd9284304c"
  >
    <li>
      <h1>
        Evacuation
      </h1>
      <img
        alt="Evacuation"
        src="http://gatherer.wizards.com/Handlers/Image.ashx?multiverseid=129541&type=card"
      />
    </li>
  </a>
  <a
    href="/9acf4d13-d68f-5fec-93b4-caf0a14725a5"
  >
    <li>
      <h1>
        Afflict
      </h1>
      <img
        alt="Afflict"
        src="http://gatherer.wizards.com/Handlers/Image.ashx?multiverseid=135206&type=card"
      />
    </li>
  </a>
  <a
    href="/4429ce97-b78e-5bc4-bd34-a38b26794bdf"
  >
    <li>
      <h1>
        Agonizing Memories
      </h1>
      <img
        alt="Agonizing Memories"
        src="http://gatherer.wizards.com/Handlers/Image.ashx?multiverseid=135228&type=card"
      />
    </li>
  </a>
  <a
    href="/bba43f4e-a9eb-5c89-8389-5935a1fbd8b3"
  >
    <li>
      <h1>
        Ascendant Evincar
      </h1>
      <img
        alt="Ascendant Evincar"
        src="http://gatherer.wizards.com/Handlers/Image.ashx?multiverseid=106525&type=card"
      />
    </li>
  </a>
  <a
    href="/c206ab94-4325-5766-91d6-62b8f50aeac8"
  >
    <li>
      <h1>
        Ascendant Evincar
      </h1>
      <img
        alt="Ascendant Evincar"
      />
    </li>
  </a>
  <a
    href="/c6f3da33-27e8-51e4-9859-ef7cb3cdbe61"
  >
    <li>
      <h1>
        Assassinate
      </h1>
      <img
        alt="Assassinate"
        src="http://gatherer.wizards.com/Handlers/Image.ashx?multiverseid=135194&type=card"
      />
    </li>
  </a>
  <a
    href="/efdd5367-9393-54c1-a6b8-c7810ee87ed5"
  >
    <li>
      <h1>
        Beacon of Unrest
      </h1>
      <img
        alt="Beacon of Unrest"
        src="http://gatherer.wizards.com/Handlers/Image.ashx?multiverseid=135270&type=card"
      />
    </li>
  </a>
  <a
    href="/ac084dab-6ac0-55fa-94b7-e009fede9012"
  >
    <li>
      <h1>
        Bog Wraith
      </h1>
      <img
        alt="Bog Wraith"
        src="http://gatherer.wizards.com/Handlers/Image.ashx?multiverseid=129491&type=card"
      />
    </li>
  </a>
  <a
    href="/efaa7c3d-c5e0-5346-b3b2-c6a3d132c27a"
  >
    <li>
      <h1>
        Bog Wraith
      </h1>
      <img
        alt="Bog Wraith"
      />
    </li>
  </a>
  <a
    href="/5ac794d2-4c66-5332-afb1-54b24bc11823"
  >
    <li>
      <h1>
        Hurkyl's Recall
      </h1>
      <img
        alt="Hurkyl's Recall"
        src="http://gatherer.wizards.com/Handlers/Image.ashx?multiverseid=135260&type=card"
      />
    </li>
  </a>
  <a
    href="/4fa9bf42-9170-594e-bbf9-42111adea313"
  >
    <li>
      <h1>
        Reminisce
      </h1>
      <img
        alt="Reminisce"
        src="http://gatherer.wizards.com/Handlers/Image.ashx?multiverseid=129698&type=card"
      />
    </li>
  </a>
  <a
    href="/f37b2caa-03c5-51ec-a02a-c2e5311a9fd5"
  >
    <li>
      <h1>
        Remove Soul
      </h1>
      <img
        alt="Remove Soul"
        src="http://gatherer.wizards.com/Handlers/Image.ashx?multiverseid=129699&type=card"
      />
    </li>
  </a>
  <a
    href="/9626b37d-3a04-5cd6-a636-943981354e2b"
  >
    <li>
      <h1>
        Robe of Mirrors
      </h1>
      <img
        alt="Robe of Mirrors"
        src="http://gatherer.wizards.com/Handlers/Image.ashx?multiverseid=134764&type=card"
      />
    </li>
  </a>
  <a
    href="/23aabc91-9146-5367-9cce-4fe955861abb"
  >
    <li>
      <h1>
        Robe of Mirrors
      </h1>
      <img
        alt="Robe of Mirrors"
      />
    </li>
  </a>
  <a
    href="/f1eba89f-dd82-5843-80a9-e9b2cd3a6037"
  >
    <li>
      <h1>
        Rootwater Commando
      </h1>
      <img
        alt="Rootwater Commando"
        src="http://gatherer.wizards.com/Handlers/Image.ashx?multiverseid=129733&type=card"
      />
    </li>
  </a>
  <a
    href="/f76ee94d-b1a1-5c9f-bdc8-4bf659844cd8"
  >
    <li>
      <h1>
        Rootwater Commando
      </h1>
      <img
        alt="Rootwater Commando"
      />
    </li>
  </a>
  <a
    href="/eee00aa2-c730-5ff1-84a9-51d95a9fe180"
  >
    <li>
      <h1>
        Rule of Law
      </h1>
      <img
        alt="Rule of Law"
        src="http://gatherer.wizards.com/Handlers/Image.ashx?multiverseid=136291&type=card"
      />
    </li>
  </a>
  <a
    href="/fc96da15-9362-585a-b555-f7a7e5480e8b"
  >
    <li>
      <h1>
        Rootwater Matriarch
      </h1>
      <img
        alt="Rootwater Matriarch"
        src="http://gatherer.wizards.com/Handlers/Image.ashx?multiverseid=135252&type=card"
      />
    </li>
  </a>
  <a
    href="/cdf65c48-2ac8-522a-bbcf-2f8165187a18"
  >
    <li>
      <h1>
        Sage Owl
      </h1>
      <img
        alt="Sage Owl"
        src="http://gatherer.wizards.com/Handlers/Image.ashx?multiverseid=135217&type=card"
      />
    </li>
  </a>
  <a
    href="/67adcf6f-bf56-5ca8-a3fb-a1045b4fba41"
  >
    <li>
      <h1>
        Sage Owl
      </h1>
      <img
        alt="Sage Owl"
      />
    </li>
  </a>
  <a
    href="/d5124438-a5d4-5b3b-88af-d179f4b9c9af"
  >
    <li>
      <h1>
        Scalpelexis
      </h1>
      <img
        alt="Scalpelexis"
        src="http://gatherer.wizards.com/Handlers/Image.ashx?multiverseid=130818&type=card"
      />
    </li>
  </a>
  <a
    href="/f4670242-6ae0-5ec4-872d-391386bb026b"
  >
    <li>
      <h1>
        Scalpelexis
      </h1>
      <img
        alt="Scalpelexis"
      />
    </li>
  </a>
  <a
    href="/be665b02-1cf2-50c6-8861-85da921bc853"
  >
    <li>
      <h1>
        Serra Angel
      </h1>
      <img
        alt="Serra Angel"
        src="http://gatherer.wizards.com/Handlers/Image.ashx?multiverseid=129726&type=card"
      />
    </li>
  </a>
  <a
    href="/9db6f50d-3242-5cbc-9f71-45477f4967df"
  >
    <li>
      <h1>
        Sea Monster
      </h1>
      <img
        alt="Sea Monster"
        src="http://gatherer.wizards.com/Handlers/Image.ashx?multiverseid=129719&type=card"
      />
    </li>
  </a>
  <a
    href="/a122a3b0-b686-5b96-a02c-926423c65b13"
  >
    <li>
      <h1>
        Shimmering Wings
      </h1>
      <img
        alt="Shimmering Wings"
        src="http://gatherer.wizards.com/Handlers/Image.ashx?multiverseid=129553&type=card"
      />
    </li>
  </a>
  <a
    href="/41b6098d-f7d9-5a37-9734-01dff7ce4912"
  >
    <li>
      <h1>
        Shimmering Wings
      </h1>
      <img
        alt="Shimmering Wings"
      />
    </li>
  </a>
  <a
    href="/39adc884-8e24-572d-ad54-74561b1cb139"
  >
    <li>
      <h1>
        Sift
      </h1>
      <img
        alt="Sift"
        src="http://gatherer.wizards.com/Handlers/Image.ashx?multiverseid=136217&type=card"
      />
    </li>
  </a>
  <a
    href="/18db55ad-e62f-5c76-a6b8-743378d6ea38"
  >
    <li>
      <h1>
        Sky Weaver
      </h1>
      <img
        alt="Sky Weaver"
        src="http://gatherer.wizards.com/Handlers/Image.ashx?multiverseid=130998&type=card"
      />
    </li>
  </a>
  <a
    href="/e451cc8a-2bbe-5850-adc4-fb1b738bcef0"
  >
    <li>
      <h1>
        Sky Weaver
      </h1>
      <img
        alt="Sky Weaver"
      />
    </li>
  </a>
  <a
    href="/889dbfab-bcc2-5a94-ba57-01d229c6dccd"
  >
    <li>
      <h1>
        Mortivore
      </h1>
      <img
        alt="Mortivore"
        src="http://gatherer.wizards.com/Handlers/Image.ashx?multiverseid=129648&type=card"
      />
    </li>
  </a>
  <a
    href="/4cd3a77f-82ac-5204-8f60-b04234d2db30"
  >
    <li>
      <h1>
        Mortivore
      </h1>
      <img
        alt="Mortivore"
      />
    </li>
  </a>
  <a
    href="/5d53a06b-68a0-5d1a-a1a5-a676f16c23dd"
  >
    <li>
      <h1>
        Nomad Mythmaker
      </h1>
      <img
        alt="Nomad Mythmaker"
        src="http://gatherer.wizards.com/Handlers/Image.ashx?multiverseid=130547&type=card"
      />
    </li>
  </a>
  <a
    href="/214dbfe9-0f16-5b6b-be30-8771192a89ec"
  >
    <li>
      <h1>
        Paladin en-Vec
      </h1>
      <img
        alt="Paladin en-Vec"
        src="http://gatherer.wizards.com/Handlers/Image.ashx?multiverseid=129668&type=card"
      />
    </li>
  </a>
  <a
    href="/284ed7cc-8917-5005-bfa9-e7c61b1fb85e"
  >
    <li>
      <h1>
        Peek
      </h1>
      <img
        alt="Peek"
        src="http://gatherer.wizards.com/Handlers/Image.ashx?multiverseid=130903&type=card"
      />
    </li>
  </a>
  <a
    href="/8d5006e3-45cb-5d9f-896d-83a22d86d53d"
  >
    <li>
      <h1>
        Phantom Warrior
      </h1>
      <img
        alt="Phantom Warrior"
        src="http://gatherer.wizards.com/Handlers/Image.ashx?multiverseid=132064&type=card"
      />
    </li>
  </a>
  <a
    href="/5a1f3fc3-e970-5408-94ef-e7c78e57000b"
  >
    <li>
      <h1>
        Plagiarize
      </h1>
      <img
        alt="Plagiarize"
        src="http://gatherer.wizards.com/Handlers/Image.ashx?multiverseid=129883&type=card"
      />
    </li>
  </a>
  <a
    href="/26180f1b-41a1-5fa5-9f6b-39ee08a1a203"
  >
    <li>
      <h1>
        Consume Spirit
      </h1>
      <img
        alt="Consume Spirit"
        src="http://gatherer.wizards.com/Handlers/Image.ashx?multiverseid=129505&type=card"
      />
    </li>
  </a>
  <a
    href="/2562f5f1-4de2-5834-a9ff-99092380d910"
  >
    <li>
      <h1>
        Contaminated Bond
      </h1>
      <img
        alt="Contaminated Bond"
        src="http://gatherer.wizards.com/Handlers/Image.ashx?multiverseid=129590&type=card"
      />
    </li>
  </a>
  <a
    href="/e77dbe8c-5878-5881-b5f3-8203af673278"
  >
    <li>
      <h1>
        Cruel Edict
      </h1>
      <img
        alt="Cruel Edict"
        src="http://gatherer.wizards.com/Handlers/Image.ashx?multiverseid=129514&type=card"
      />
    </li>
  </a>
  <a
    href="/7069fe50-d855-59ea-b445-e8068cda7559"
  >
    <li>
      <h1>
        Deathmark
      </h1>
      <img
        alt="Deathmark"
        src="http://gatherer.wizards.com/Handlers/Image.ashx?multiverseid=129910&type=card"
      />
    </li>
  </a>
  <a
    href="/e9435188-40bd-54c7-938f-798459b038ed"
  >
    <li>
      <h1>
        Diabolic Tutor
      </h1>
      <img
        alt="Diabolic Tutor"
        src="http://gatherer.wizards.com/Handlers/Image.ashx?multiverseid=129525&type=card"
      />
    </li>
  </a>
  <a
    href="/77ce01d6-3609-519e-a0d4-6ad6770c45e4"
  >
    <li>
      <h1>
        Distress
      </h1>
      <img
        alt="Distress"
        src="http://gatherer.wizards.com/Handlers/Image.ashx?multiverseid=135219&type=card"
      />
    </li>
  </a>
  <a
    href="/49f64f3e-f0ff-5d4c-a730-5ea919857d82"
  >
    <li>
      <h1>
        Doomed Necromancer
      </h1>
      <img
        alt="Doomed Necromancer"
        src="http://gatherer.wizards.com/Handlers/Image.ashx?multiverseid=129880&type=card"
      />
    </li>
  </a>
  <a
    href="/fea2ad06-355d-5a9a-822d-66275d89cad9"
  >
    <li>
      <h1>
        Dross Crocodile
      </h1>
      <img
        alt="Dross Crocodile"
        src="http://gatherer.wizards.com/Handlers/Image.ashx?multiverseid=135216&type=card"
      />
    </li>
  </a>
  <a
    href="/2d080e75-7e1b-5f75-9c8f-18d02a4710dd"
  >
    <li>
      <h1>
        Drudge Skeletons
      </h1>
      <img
        alt="Drudge Skeletons"
        src="http://gatherer.wizards.com/Handlers/Image.ashx?multiverseid=129529&type=card"
      />
    </li>
  </a>
  <a
    href="/42022e6f-e123-54a6-84df-ec826b13f5be"
  >
    <li>
      <h1>
        Dusk Imp
      </h1>
      <img
        alt="Dusk Imp"
        src="http://gatherer.wizards.com/Handlers/Image.ashx?multiverseid=129490&type=card"
      />
    </li>
  </a>
  <a
    href="/b75509f6-c893-54a7-a51e-757066a0b4da"
  >
    <li>
      <h1>
        Dusk Imp
      </h1>
      <img
        alt="Dusk Imp"
      />
    </li>
  </a>
  <a
    href="/d38298f2-29e8-57b0-b978-140aa2171cce"
  >
    <li>
      <h1>
        Essence Drain
      </h1>
      <img
        alt="Essence Drain"
        src="http://gatherer.wizards.com/Handlers/Image.ashx?multiverseid=135191&type=card"
      />
    </li>
  </a>
  <a
    href="/b159e707-915c-5940-bd70-7f0416cc8def"
  >
    <li>
      <h1>
        Fear
      </h1>
      <img
        alt="Fear"
        src="http://gatherer.wizards.com/Handlers/Image.ashx?multiverseid=129544&type=card"
      />
    </li>
  </a>
  <a
    href="/6d030b32-0534-55c9-9c40-00bb59d4e164"
  >
    <li>
      <h1>
        Hidden Horror
      </h1>
      <img
        alt="Hidden Horror"
        src="http://gatherer.wizards.com/Handlers/Image.ashx?multiverseid=135234&type=card"
      />
    </li>
  </a>
  <a
    href="/60122d6f-448b-5df8-ac2a-8f5a1e841278"
  >
    <li>
      <h1>
        Nantuko Husk
      </h1>
      <img
        alt="Nantuko Husk"
        src="http://gatherer.wizards.com/Handlers/Image.ashx?multiverseid=129653&type=card"
      />
    </li>
  </a>
  <a
    href="/f560811b-3c5d-5929-8b17-e9ae8730cec1"
  >
    <li>
      <h1>
        Nekrataal
      </h1>
      <img
        alt="Nekrataal"
        src="http://gatherer.wizards.com/Handlers/Image.ashx?multiverseid=129658&type=card"
      />
    </li>
  </a>
  <a
    href="/37f0071d-09b9-5fdc-a5de-52061a01cb64"
  >
    <li>
      <h1>
        Nekrataal
      </h1>
      <img
        alt="Nekrataal"
      />
    </li>
  </a>
  <a
    href="/5bd291f0-560a-5046-bd07-a91bd3004f2d"
  >
    <li>
      <h1>
        Nightmare
      </h1>
      <img
        alt="Nightmare"
        src="http://gatherer.wizards.com/Handlers/Image.ashx?multiverseid=129659&type=card"
      />
    </li>
  </a>
</ul>
`;
